### ISSUE TITLE
Crash in Time Gentemen Please and Ben There, Done That

### ISSUE THREAD

**Issue by dreammaster** [adventuregamestudio/ags#1412]
AGS has crashes for both the commercial games Time Gentelemen Please and Ben There Dan That, as reported in https://bugs.scummvm.org/ticket/12922

The easiest to reproduce is the crash in the intro of Time Gentlemen Please. It seems to be from calling SetObjectFrame with a value of -1. I tried changing the error line to a warning, but then getting further secondary errors. Hopefully you'll be able to better handle on how to properly work around the problems.

**Comment by ivan-mogilko** [comment]
@dreammaster Apparently I made the rules stricter again in recent version. Checking older code, engine had always allocated 1 dummy frame for the empty view loops, and object could fallback to that dummy frame in case script command told it to assign that empty loop.

The dummy frame references sprite 0.

So I just restored that behavior now (commit 1496f7f6419d192bd4060956492e8897678035c9, also preceded by 095db88ae0f1acceb298f793a2e1b81dbe06fcff but idk if this one will be useful to scummvm).

I think the game eventually changes an object to something else, because otherwise it would look wrong.
